monitor.py

```diff
@@ -258,10 +269,9 @@ def check_disk_space(paths):
 
 
 def poll_online_workers():
+    public_workers = get_public_workers()
     if len(public_workers) == 0:
-        error_logs(
-            'worker check failed', 'Missing value for environment variable CODALAB_PUBLIC_WORKERS.'
-        )
+        log("Environment variable CODALAB_PUBLIC_WORKERS was not configured.")
```

Comment:
Nit: Can we change this simply to CODALAB_PUBLIC_WORKERS is empty?  It might have been configured to be empty, technically.